concepts/string.md

```diff
@@ -0,0 +1,36 @@
+---
+description: String
+---
+
+# String
+
+A string is an object in memory that stores a sequence of characters and colloquially it is the representation of text in the computer.
+Examples:
+1. `bicycle`
+2. `A man with a telescope 🔭 from the city of Třebíč`
+3. ```
+   Introduction: Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+   
+   Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
+   ```
+
+The first example is simply a 7-letter word of English letters.
```

Comment:
Let's avoid this type of long-term dependency - pointers are hard to read and hard to maintain.

It's a lot of examples and text to make a few basic points.

> A string can be short or even empty.  A string with no characters is the *empty string*.
> > `a`
>
> A string can be long or even infinite.
> `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. ...`
> 
> A string can contain any character, including control characters like *newline*.
> ```
> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
>
> Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
> ```
> 
> By default, a string is *plain text*.  It does not represent rich formatting or styles, like font-size and color.

Reply:
> A string can be long or even infinite
:-)

Updated the examples.